torch/nativert/common/Conv.cpp

```diff
@@ -0,0 +1,51 @@
+#include <torch/nativert/common/Conv.h>
+
+#include <cerrno>
+#include <cstdlib>
+#include <string>
+
+namespace torch::nativert {
+
+template <>
+std::optional<int64_t> tryTo<int64_t>(std::string_view symbol) {
```

Comment:
I would expect fmt has such functions?
@swolchok or @cyyever might know off the top of their head

Reply:
[std::stoll](https://en.cppreference.com/w/cpp/string/basic_string/stol) seems likely to be nicer than making use of bare strtoll. There is also [std::from_chars](https://en.cppreference.com/w/cpp/utility/from_chars) but apparently libc++ didn't support it as recently as last year. (Disclaimer: I haven't used either of these, but I would try.)

Reply:
@swolchok I tried std::from_chars but ran into some weird compile errors from MSVC. :( I can give it another try on stoll.

Reply:
Hmm I'm still inclined to use strtoll. std::stoll seems to throw exception and we expect the nullopt case for this function.